app/views/provider_mailer/organisation_permissions_set_up.text.erb

```diff
@@ -0,0 +1,28 @@
+Dear <%= @provider_user.full_name %>
+
+# <%= @partner_organisation.name %> has set up organisation permissions for teacher training courses you work on with them
+
+You can now manage applications made through GOV.UK for the courses you work on with <%= @partner_organisation.name %>.
+
+They have set up the following organisation permissions.
+
+Make offers and reject applications:
+
+<%= "- #{@recipient_organisation.name}" if @permissions.permit?(:make_decisions, @recipient_organisation) %>
+<%= "- #{@partner_organisation.name}" if @permissions.permit?(:make_decisions, @partner_organisation) %>
+
+View criminal convictions and professional misconduct:
+
+<%= "- #{@recipient_organisation.name}" if @permissions.permit?(:view_safeguarding_information, @recipient_organisation) %>
+<%= "- #{@partner_organisation.name}" if @permissions.permit?(:view_safeguarding_information, @partner_organisation) %>
+
+View sex, disability and ethnicity information:
+
+<%= "- #{@recipient_organisation.name}" if @permissions.permit?(:view_diversity_information, @recipient_organisation) %>
+<%= "- #{@partner_organisation.name}" if @permissions.permit?(:view_diversity_information, @partner_organisation) %>
+
+# Change organisation permissions
+
+You can change these organisation permissions in your organisation settings:
+
+<%= provider_interface_edit_provider_relationship_permissions_url(@permissions) %>
```

Comment:
Since this email is going out with the feature flag, can we point this to the organisation settings page (same for the other email too) as we're removing those old routes shortly. Not sure if we want to point to /organisation-settings, or the org permissions page for a specific provider though

Reply:
Ah yes I wasn't sure about this link, the wording in the paragraph above it suggests it's organisation settings. Will fix.

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5235/commits/0640d6febc7bb6180561eaf9f0d52e14039738eb